### PR TITLE
Authz android quickstart

### PR DESCRIPTION
**Pull request by jhiner**
Submitted for review, this is the Android OAuth 2.0 API Quickstart.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by jhiner** [comment]
cc @cabralmartin @hzalaz

File: articles/native-platforms/android/_includes/_api_authz.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+public void startAuth() {
+
+    Map<String, Object> params = new HashMap<String, Object>();
+    params.put("audience", "{YOUR API IDENTIFIER}");
+
+    WebAuthProvider.init(account)
+            .withConnection("Username-Password-Authentication")
+            .withScope("openid profile {API SCOPES}")
+            .withParameters(params)
+            .useCodeGrant(true)
```
Comment (hzalaz) [inline articles/native-platforms/android/_includes/_api_authz.md]: I think this is the default so it's safe to remove it

File: articles/native-platforms/android/_includes/_api_authz.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    super.onNewIntent(intent);
+}
+```
+
+## Making an Authenticated API Call
+
+Use the `access_token` to invoke your Resource Server (API):
+
+```java
+HttpResponse<String> response = Unirest.get("https://someapi.com/api")
```
Comment (hzalaz) [inline articles/native-platforms/android/_includes/_api_authz.md:121]: Mention you are using an external lib

**Review by hzalaz — CHANGES_REQUESTED** [review]

File: articles/native-platforms/android/_includes/_api_authz.md (hunk 1 above)
Comment (jhiner) [reply articles/native-platforms/android/_includes/_api_authz.md]: fixed.

File: articles/native-platforms/android/_includes/_api_authz.md (hunk 2 above)
Comment (jhiner) [reply articles/native-platforms/android/_includes/_api_authz.md:121]: fixed.

**Review by hzalaz — APPROVED** [review]

**Comment by cabralmartin** [comment]
Ship it!

**Pull request merged**